packages/transition-backend/src/services/transitRouting/TrRoutingBatch.ts

```diff
@@ -196,7 +188,7 @@ class TrRoutingBatch {
             this.options.progressEmitter.emit('progress', { name: 'BatchRouting', progress: 1.0 });
             this.options.progressEmitter.emit('progress', { name: 'StoppingRoutingParallelServers', progress: 0.0 });
 
-            const stopStatus = await TrRoutingProcessManager.stopMultiple(trRoutingInstancesCount);
+            const stopStatus = await TrRoutingProcessManager.stopBatch();
```

Comment:
Pass port number

Reply:
We started with the default, so I stop with the default.